cumulus/parachains/runtimes/contracts/contracts-rococo/src/lib.rs

```diff
@@ -100,7 +100,7 @@ pub type UncheckedExtrinsic =
 /// Migrations to apply on runtime upgrade.
 pub type Migrations = (
 	cumulus_pallet_parachain_system::migration::Migration<Runtime>,
-	cumulus_pallet_xcmp_queue::migration::MigrationToV3<Runtime>,
```

Comment:
I think we either have to keep all of them in, or remove all.  
And please always put new migrations at the end.

Reply:
Done